MekHQ/resources/mekhq/resources/CampaignGUI.properties

```diff
@@ -75,23 +75,19 @@ miCargoReport.text=Cargo Report...
 # View Menu
 menuView.text=View
 miShowHistoricalReportLog.text=Historical Daily Report Log
-miDetachLog.text=Detach Daily Report Log
-miAttachLog.text=Attach Daily Report Log
```

Comment:
We really ought to look at making the daily log detachable again. It's such a pain in the ass to have to keep going back to the command center tab for it.

Reply:
I concur, and will open up an issue to open it in its own modeless window.